### ISSUE TITLE
Exception when getting token from session client

### ISSUE THREAD

**Issue by bedirguven** [okta/okta-oidc-android#151]
Calling getTokens from SessionClient will cause : 

```
 E/UncaughtException: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String[] java.lang.String.split(java.lang.String)' on a null object reference
        at com.okta.oidc.Tokens.<init>(Tokens.java:65)
        at com.okta.oidc.clients.sessions.SyncSessionClientImpl.getTokens(SyncSessionClientImpl.java:205)
        at com.okta.oidc.clients.sessions.SessionClientImpl.getTokens(SessionClientImpl.java:120)
```

After OktaState call getTokenResponse then response has null value for scope and new token not creating. So i can't get 'access_token' because of this exception.
```

@Override
    public Tokens getTokens() throws AuthorizationException {
        try {
            TokenResponse response = mOktaState.getTokenResponse();
            if (response == null) {
                return null;
            }
            return new Tokens(response);
        } catch (OktaRepository.EncryptionException e) {
            throw AuthorizationException.EncryptionErrors.byEncryptionException(e);
        }
    }
```

```
@RestrictTo(RestrictTo.Scope.LIBRARY)
    public Tokens(@NonNull TokenResponse response) {
        this(response.getIdToken(), response.getAccessToken(),
                response.getRefreshToken(), Integer.parseInt(response.getExpiresIn()),
                response.getScope().split(" "), response.getExpiresAt());
    }
```

Is there any solution for this?

**Comment by FeiChen-okta** [comment]
@bedirguven are you still having issues with this? The token response has no scope?

**Comment by bedirguven** [comment]
@FeiChen-okta Yes i do

**Comment by bedirguven** [comment]
> @bedirguven are you still having issues with this? The token response has no scope?

Yes i fo

**Comment by FeiChen-okta** [comment]
Hi @bedirguven you might want to check your server to see if you have the scopes set correctly. The response should contain a scope. If it doesn't you shouldn't even have an access token. You can check your authorization servers scope: https://developer.okta.com/docs/guides/customize-authz-server/create-scopes/
Check if the server you are using have the same scope you've set in the client.

**Comment by patluri43** [comment]
@bedirguven I am seeing the same issue, It used to work earlier with the same version of SDK, Not sure what changed

**Comment by bedirguven** [comment]
@FeiChen-okta https://www.oauth.com/oauth2-servers/access-tokens/access-token-response/ for this documentation as you can see, if the scope the user granted is identical to the scope the app requested, scope parameter is optional.We are using same scope with server in the client. So we don't have scope data that we got from server.But in the okta library token parse function have required scope parameter and that parameter can be null.So this is the crash reason.

**Comment by FeiChen-okta** [comment]
Hi @bedirguven Yes the issue is that it is assuming an oidc authorization server. which should contain at least openid scope. Will file bug to make the change to support oauth2 server.

**Comment by FeiChen-okta** [comment]
Hi @bedirguven This is fixed with the 1.0.14 release.
